src/main/java/teammates/ui/controller/AdminHomePageData.java

```diff
@@ -8,7 +8,8 @@
     public String instructorName;
     public String instructorEmail;
     public String instructorInstitution;
-
+    public String instructorDetails;
```

Comment:
If possible, try to think of a better name for instructorDetails, or leave a small comment here. When reading through the code, it's not very apparent what instructorDetails is supposed to be.